_data/simple-icons.json

```
@@ -2374,6 +2374,11 @@
             "hex": "F7DF1E",
             "source": "https://github.com/cytoscape/cytoscape.js/blob/97c27700feefe2f7b79fca248763049e9a0b38c6/documentation/img/cytoscape-logo.svg"
         },
+        {
+            "title": "D",
+            "hex": "B03931",
+            "source": "https://github.com/dlang/dlang.org/blob/master/images/dlogo.svg"
```

Please use a permalink to the logo file as described in the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md):

> In any case, the commit hash should be part of the URL. On GitHub, you can get the correct URL by pressing `y` on the GitHub page you want to link to. You can get help at the [getting permanent links to files page](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files).

```suggestion
            "source": "https://github.com/dlang/dlang.org/blob/6d0e2e5f6a8249031cfd010e389b99ff014cd320/images/dlogo.svg"
```

---

Let's retain quality issues. If we want to fix those, they should be upstreamed first.